user-org-actor/src/main/java/org/sunbird/organization/OrgReadActor.java

```diff
@@ -0,0 +1,24 @@
+package org.sunbird.organization;
+
+import org.sunbird.BaseActor;
+import org.sunbird.actor.core.ActorConfig;
+import org.sunbird.actorOperation.OrgActorOperations;
+import org.sunbird.organization.service.IOrgService;
+import org.sunbird.organization.service.OrgServiceImpl;
+import org.sunbird.request.Request;
+
+@ActorConfig(
+  tasks = {"readOrg"},
+  asyncTasks = {}
+)
+public class OrgReadActor extends BaseActor {
+
+  private static IOrgService INSTANCE = new OrgServiceImpl();
+
+  @Override
+  public void onReceive(Request request) {
+    if (OrgActorOperations.READ_ORG.getOperation().equalsIgnoreCase(request.getOperation())) {
```

Comment:
Can you create a class and put this as static method?
OperationValidator.checkName(request.getOperation, OrgActorOperations.READ_ORG)

We can burry this equalsIgnoreCase there...I dislike that condition and maybe it will be hidden from Actor that way :-)

Also, see that this check is different in OrgCreateActor. Am wondering if there is any real good reason this must exist.

Reply:
done